controllers/apps/components/stateless.go

```diff
@@ -279,3 +281,14 @@ func (c *statelessComponent) updateWorkload(deployObj *appsv1.Deployment) {
 		c.SetStatusPhase(appsv1alpha1.SpecReconcilingClusterCompPhase, nil, "Component workload updated")
 	}
 }
+
+func (c *statelessComponent) updatePodEnvConfig() {
```

Comment:
do assure code coverage

Reply:
noted with thanks.  will add tests for components sometime later.